boards/adi/max32657evkit/doc/index.rst

```diff
@@ -343,6 +343,171 @@ using :zephyr:code-sample:`blinky` sample:
    :goals: build
 
 
+BOARD: max32657evkit/max32657/ns
+================================
+
+The ``max32657evkit/max32657/ns`` board configuration is used to build the secure firmware
```

Comment:
board target, no such term as board configuration in zephyr

Reply:
Added to classify them to simplify readability. I can update it if require?

- BOARD: max32657evkit/max32657
- BOARD: max32657evkit/max32657/ns